devito/ir/iet/visitors.py

```diff
@@ -524,8 +530,12 @@ def visit_Iteration(self, o):
     def visit_Expression(self, o):
         return filter_sorted([f for f in self.rule(o)], key=attrgetter('name'))
 
+    def visit_Call(self, o):
+        symbols = self._visit(o.children)
+        symbols += [f for f in self.rule(o)]
```

Comment:
symbols.append(...)

Reply:
That wouldn't work. The alternative would be `extend` but it's equivalent